README.md

```diff
@@ -1,6 +1,6 @@
 # LetMeIn v2 <img src="https://forthebadge.com/images/badges/you-didnt-ask-for-this.svg" alt="ohio badge" height="30px"/>
 
-<img src="https://csh.rit.edu/~wilnil/storage/of-ohioan-descent.svg" alt="ohio badge" height="30px"/> 
+<!-- <img src="https://csh.rit.edu/~wilnil/storage/of-ohioan-descent.svg" alt="ohio badge" height="30px"/>  -->
```

Comment:
If you're commenting something out, potentially remove it?

Reply:
Sugma